bounceview/src/main/java/com/github/hariprasanths/bounceview/BounceView.java

```diff
@@ -161,55 +161,8 @@ public BounceViewAnim setInterpolatorPopOut(Animator interpolatorPopOut) {
 
     private void setAnimToView() {
         if (mComponent != null) {
-            mComponent.get().setTouchEventListener((component, motionEvent) -> {
-                int action = motionEvent.getAction();
-                if (action == TouchEvent.PRIMARY_POINT_DOWN) {
-                    isTouchInsideView = true;
-                    startAnimScale(component, pushInScaleX, pushInScaleY, pushInAnimDuration, pushInInterpolator, 0);
-                    return true;
-                } else if (action == TouchEvent.PRIMARY_POINT_UP) {
-                    if (isTouchInsideView) {
-                        component.createAnimatorProperty().cancel();
-                        startAnimScale(component, popOutScaleX, popOutScaleY,
-                                popOutAnimDuration, popOutInterpolator, 0);
-                        startAnimScale(component, 1f, 1f, popOutAnimDuration,
-                                popOutInterpolator, popOutAnimDuration + 1);
-                        return false;
-                    }
-                } else {
-                    return checkAction(action, component, motionEvent);
-                }
-                return false;
-            });
-        }
-    }
-
-    private boolean checkAction(int action, Component component, TouchEvent motionEvent) {
-        if (action == TouchEvent.CANCEL) {
-            if (isTouchInsideView) {
-                component.createAnimatorProperty().cancel();
-                startAnimScale(component, 1f, 1f, popOutAnimDuration, DEFAULT_INTERPOLATOR, 0);
-            }
-            return true;
-        } else if (action == TouchEvent.POINT_MOVE && isTouchInsideView) {
-            int i = motionEvent.getIndex();
-            float currentX = motionEvent.getPointerScreenPosition(i).getX();
-            float currentY = motionEvent.getPointerScreenPosition(i).getY();
-            float currentPosX = currentX + component.getLeft();
-            float currentPosY = currentY + component.getTop();
-            float viewLeft = component.getLeft();
-            float viewTop = component.getTop();
-            float viewRight = component.getRight();
-            float viewBottom = component.getBottom();
-            if (!(currentPosX > viewLeft && currentPosX < viewRight
-                    && currentPosY > viewTop && currentPosY < viewBottom)) {
-                isTouchInsideView = false;
-                component.createAnimatorProperty().cancel();
-                startAnimScale(component, 1f, 1f, popOutAnimDuration, DEFAULT_INTERPOLATOR, 0);
-            }
-            return true;
+            check(mComponent.get());
```

Comment:
rename the method to setListenerForTouchEvents

Reply:
done